templates/layouts/_questionnaire.html

```diff
@@ -39,4 +42,22 @@
     {% endif %}
   {% endblock %}
 
+  {{
+    onsTimeoutModal({
+      "showModalTimeInSeconds": 60,
+      "serverSessionExpiryEndpoint": "/session-expiry",
+      "sessionExpiresAt": "",
```

Comment:
The expires at will need to be exposed to the templates and will need passing in here.

Reply:
So I made it part of cookies as "expires_at" and pass it to this bit of template as part of `session`. It gets updated whenever session is extended. Thanks!